tests/robot-tests/tests/admin/bau/comments.robot

```diff
@@ -79,21 +79,16 @@ Navigate to content section as bau1
 
 Add review comment for first content block
     [Tags]    HappyPath
-    user scrolls to element    testid:accordionSection
     user clicks button    First content section
-
-    user scrolls to element    testid:Expand Details Section Add / View comments (0 unresolved)
-    user clicks element    testid:Expand Details Section Add / View comments (0 unresolved)
-
-    # avoid set page view box getting in the way
-    user scrolls down    400
+    user clicks button    Edit block
+    # select the text to enable the add comment button
+    user presses keys    CTRL+a
+    user clicks element    xpath://*[@aria-label="Editor toolbar"]//button[9]    # CKEditor comment button
```

Comment:
I'd be tempted to try to use a more specific selector here if we can (no probs if not)

Reply:
I couldn't get a more specific selector to work here. The test that clicks the link button uses this pattern so I ended up copying it here.